apps/backend/api/graphql/mutations/event.js

```diff
@@ -17,11 +18,42 @@ export async function respondToEvent (userId, eventId, response) {
   } else {
     await EventInvitation.create({
       userId,
-      inviterId: userId,
+      inviterId: userId, // why is the user who is responding also the inviter? Is this a workaround?
       eventId,
       response
     })
   }
+
+  if (response === EventInvitation.RESPONSE.YES ||
+    response === EventInvitation.RESPONSE.INTERESTED
+  ) {
+    const iCalendar = ical()
+    const iCalData = await event.getCalData(userId)
+    iCalendar.createEvent(iCalData)
```

Comment:
so the only thing im now realizing is that this will slow down the mutation a bit, if it still seems very fast then this is fine, but the other option is to move all this generation into the Email.sendEventRsvpEmail. i'll leave that decision to you.